repository/repository.go

```diff
@@ -25,6 +26,8 @@ var repoToOrg = map[string]string{
 	"k3s":  "k3s-io",
 }
 
+var backportRegExp = regexp.MustCompile(`^\s*\[[Rr]elease-?\s*[\w\d\.]*\]\s*(.*)$`)
```

Comment:
What all is this doing?

Reply:
Added a clarifying helper func and tests ea2ccc7